.github/workflows/build-and-push-wasm-plugin-image.yaml

```diff
@@ -34,121 +100,102 @@ jobs:
       GO_VERSION: 1.19
       TINYGO_VERSION: 0.28.1
       ORAS_VERSION: 1.0.0
+
     steps:
-      - name: Set plugin_type, plugin_name and version from inputs or ref_name
+      - name: Set PLUGIN_NAME from matrix
+        run: echo "PLUGIN_NAME=${{ matrix.plugin }}" >> $GITHUB_ENV
+
+      - name: Determine plugin_type, builder_image and version  # 设置插件类型、构建器镜像、版本
         id: set_vars
         run: |
           if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
             plugin_type="${{ github.event.inputs.plugin_type }}"
-            plugin_name="${{ github.event.inputs.plugin_name }}"
-            version="${{ github.event.inputs.version }}"
+            plugin_name="${{ github.event.inputs.plugin_name }}"   
+            echo "PLUGIN_NAME=$plugin_name" >> $GITHUB_ENV 
```

Comment:
```suggestion
```
These two lines are redundant because `PLUGIN_NAME` has already been set earlier using echo `"PLUGIN_NAME=${{ matrix.plugin }}" >> $GITHUB_ENV` in the matrix step.